lib/vanagon/platform/windows.rb

```diff
@@ -103,12 +104,14 @@ def generate_nuget_package(project)
       # @return [Array] list of commands required to build an msi package for the given project from a tarball
       def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
-        cg_name = "compfiles"
+        cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/staging",
-        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/staging' --strip-components 1 -xf -",
+        "mkdir -p $(tempdir)/#{project.name}/staging",
```

Comment:
It looks like you lost the changes merged in with https://github.com/puppetlabs/vanagon/pull/273!